### DISCUSSION TITLE
Übersichtlichkeit bei Blockprogrammierung

### DISCUSSION THREAD

**Discussion by msw-home** [pybricks/support#1435] [Q&A]
Moin zusammen,
ich war früher in der SPS (Speicher programmieren Steuerung) zu Hause.
Dort war es üblich, ein Programm in sogenannten Netzwerken (NW) zu programmieren.
1. NW z,B. Zuweisung der einzelnen Komponenten.
2. NW Bewegung vor
3. NW Bewegung zurück
4. usw.
Man konnte NW einfügen, lösen, duplizieren usw.
Die NW befanden sich in einem Ablauf Programmstein, diesen konnte man durchblättern.
Ich empfand das als sehr hilfreich und übersichtlich.
Ist so etwas unter Pybricks (Blöcke) auch möglich?

**Comment by ggramlich** [comment]
@msw-home Kannst du mal ein Beispiel geben (Screenshot, Skizze oder Doku)?
Ich kann mir nach deiner Beschreibung nicht vorstellen, wie das aussehen soll.

**Reply by msw-home** [reply]
Es folgt noch ein Beispiel, meine Vorstellung.

![image](https://github.com/pybricks/support/assets/155910298/31c8c81b-322a-42c4-bfb5-740e4f11034f)

**Reply by msw-home** [reply]
Hier meine Vorstellung, wie ich aus der SPS Programmierung her kenne.

![image](https://github.com/pybricks/support/assets/155910298/6ad47335-c29f-4252-8c04-0b817d54d96f)
